index.js

```diff
@@ -24,11 +24,25 @@ function forEach(xs, fn) {
 }
 
 // TODO: use object-keys
-var objectKeys = Object.keys || function keys(obj) {
+function objectKeys(obj) {
 	var res = [];
-	for (var key in obj) { res.push(key); } // eslint-disable-line no-restricted-syntax
+
+	for (var key in obj) { // eslint-disable-line no-restricted-syntax
+		res.push(key);
+	}
```

Comment:
if Object.keys is available, it should be used, and it should be cached at module level.

Reply:
Updated.